OrbitClientServices/ProcessClient.cpp

```diff
@@ -48,8 +48,10 @@ ErrorMessageOr<std::vector<orbit_grpc_protos::ProcessInfo>> ProcessClient::GetPr
 
   grpc::Status status = process_service_->GetProcessList(context.get(), request, &response);
   if (!status.ok()) {
-    ERROR("gRPC call to GetProcessList failed: %s (error_code=%d)", status.error_message(),
-          status.error_code());
+    if (status.error_code() != 4) {
```

Comment:
Doesn't that "4" have an associated constant that we could use instead? Otherwise this is a bit unclear.

Reply:
Should be `grpc::DEADLINE_EXCEEDED`.

Reply:
Or `grpc::StatusCode::DEADLINE_EXCEEDED` for better clarity, but it's not an `enum class`.

Reply:
done.